primeqa/ir/README.md

```diff
@@ -150,12 +150,12 @@ Using a model trained as described [here](https://github.com/primeqa/primeqa/tre
 python primeqa/ir/run_ir.py \
     --engine_type DPR \
     --do_index \
-    --dpr_ctx_encoder_path <context_encoder_model> \
+    --ctx_encoder_name_or_path <context_encoder_model> \
     --embed <part_number>of<parts_total> \
     --sharded_index \
```

Comment:
Can this be left default?

Reply:
The point here is to show how to use a model trained in the previous (training) step.